src/databricks/labs/ucx/runtime.py

```diff
@@ -24,6 +24,104 @@
 logger = logging.getLogger(__name__)
 
 
+def _get_view_definition(hms_db) -> str:
+    return f"""CREATE
+OR REPLACE VIEW {hms_db}.vw_failure_summary AS WITH failuretab (failure, issue, object_type) AS (
```

Comment:
```suggestion
OR REPLACE VIEW {hms_db}.failure_summary AS WITH failuretab (failure, issue, object_type) AS (
```
No need to prefix views with vw